src/Math-Tests-Matrix/PMMatrixTest.class.st

```diff
@@ -475,7 +475,10 @@ PMMatrixTest >> testMatrixInitialize [
 	| a |
 	a := PMMatrix new initialize: 2.
 	self assert: a numberOfRows equals: 2.
-	self assert: a numberOfColumns equals: 2
+	self assert: a numberOfColumns equals: 2.
+	
+	self should: [ PMMatrix new: -2 ] raise: AssertionFailure .
```

Comment:
These, according to the project guidelines, should be a separate test method e.g. `testMatrixInitialisationWithNegativeDimensions` as this is an edge case.